org.adempiere.ui.zk/WEB-INF/src/web/theme/default/css/fragment/gadget.css.dsp

```diff
@@ -216,6 +216,10 @@
 	width: 100%;
 }
 
+.dashboard-row .chart-gadget {
+	max-height: 300px; 
```

Comment:
for hardcoded height, it is better to define it by screen size, i.e "@media" notation with max or min height.

Reply:
Media query added, now everything larger than 768px (tablet to desktop) has suggested max-height 300px, and smaller devices will have 150px. It is hard to test for smaller device because I have charts with JFreeChart, for large devices we were able to test with billboard plug-in, it should work fine there.